### PR TITLE
Add build for mariadb 1.0.7 on mac

### PR DESCRIPTION
**Pull request by 9a24f0**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Fetch source distribution
+        run: |
+          curl https://files.pythonhosted.org/packages/4e/6d/0c9bf9668da4e86417f73573353d3659d222ef76a93ce1717fb33284cdea/mariadb-1.0.7.tar.gz -o mariadb-1.0.7.tar.gz
+          tar -zxf mariadb-1.0.7.tar.gz
+          mv mariadb-1.0.7/* .
+
+      - name: Build wheels
+        run: python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_BEFORE_ALL_MACOS: brew install mariadb
```
Comment (Huy-Ngo) [inline .github/workflows/build.yml]:
```suggestion
        env:
          CIBW_PROJECT_REQUIRES_PYTHON: ">=3.7"
          CIBW_BUILD: "cp*"
          CIBW_BEFORE_ALL_MACOS: brew install mariadb
```

**Review by Huy-Ngo — CHANGES_REQUESTED** [review]
Please also add checksum using my action (see README in main branch)

File: .github/workflows/build.yml (hunk 1 above)
Comment (9a24f0) [reply .github/workflows/build.yml]: Resolved as dcd7c468e9dd77df42d6d8dcc385b3618bc94a9c

**Comment by Huy-Ngo** [comment]
No need to rebase and force push :sweat: we won't merge this anyway

**Review by Huy-Ngo — APPROVED** [review]

**Comment by 9a24f0** [comment]
> No need to rebase and force push 😓 we won't merge this anyway

It's just looks more cleaner imho 😆

**Comment by Huy-Ngo** [comment]
OK, this works, but it can't release because you're not a member of the org, so I'll close this. Thanks for your contribution.

**Pull request closed without merging**